CMakeLists.txt

```diff
@@ -59,13 +59,16 @@ if(MSVC)
   endif()
 endif()
 
+include(GNUInstallDirs)
+include(CMakePackageConfigHelpers)
+
 add_library(${PROJECT_NAME} SHARED ${MODELS_SOURCES} ${OPERATOR_SOURCES} ${IMAGE_SOURCES})
 target_link_libraries(${PROJECT_NAME} PRIVATE ${TORCH_LIBRARIES} ${PNG_LIBRARY} ${JPEG_LIBRARIES} Python3::Python)
-set_target_properties(${PROJECT_NAME} PROPERTIES EXPORT_NAME TorchVision)
+set_target_properties(${PROJECT_NAME} PROPERTIES
+  EXPORT_NAME TorchVision
+  INSTALL_RPATH ${TORCH_INSTALL_PREFIX}/lib)
```

Comment:
I don't think the INSTALL_RPATH does anything on windows.
As to whether this works on osx, I am admittedly not sure (and I have no way to check). I think we should probably add a way to test this (maybe another cpp example that gets automatically run?)

Reply:
> As to whether this works on osx, I am admittedly not sure (and I have no way to check)

I can check this.

> I think we should probably add a way to test this (maybe another cpp example that gets automatically run?)

I agree.

On the other hand, as far as C++ frontend should be built by user for the same machine, this would work, right.

Reply:
> I can check this.

That would be great!

> I agree.

I have created a test in PR #2807 (still needs integration with CI) that fails for me if this PR isn't merged, so I guess that should be enough to ensure that this change works for all configurations tested in the CI

Reply:
@vfdev-5 Did you get around to checking if this patch works correctly on osx?

Reply:
@bmanga sorry, haven't got time to check it with the release. Let me check it tomorrow.